wynnset/tapestry-wp
# Show favourites in spaceship cockpit

Closes #498

**Testing**

* Create two modules with a few topics each
* Favourite several topics from each module
* Open the cockpit and click the favourites tab on the upper right corner
* The favourites from *both* modules should appear
* Open a module summary for one of the modules
* Only the favourites from that model should appear in the favourites tab
